kcidb/test_oo.py

```diff
@@ -456,3 +488,67 @@ def test_traversing_revision_links(self):
         self.assertFalse(revision.builds_valid)
 
         self.assertIsInstance(revision.tests_root, Node)
+        self._traversing_revision_root_test_node(revision)
```

Comment:
This doesn't have to be a part of this test case. We can have that as a separate, explicit test case. So please remove this call, and make that function a test case.